lib/topological_inventory/satellite/operations/source.rb

```diff
@@ -11,6 +11,14 @@ class Source
         include Logging
         STATUS_AVAILABLE, STATUS_UNAVAILABLE = %w[available unavailable].freeze
 
+        ERROR_MESSAGES = {
+          :endpoint_not_found           => "Endpoint not found in Sources API",
+          :receptor_network_unreachable => "Receptor network unreachable",
+          :receptor_node_disconnected   => "Receptor node is disconnected",
+          :receptor_node_not_defined    => "Receptor node not defined in Sources API",
+          :receptor_not_responding      => "Receptor is not responding",
```

Comment:
@agrare there was some idea on meeting that errors when updating Source's availability should be standardized (to be able to link to documentation). 
These error messages (except messages coming from Satellite's plugin) are probably nowhere specified. Should we consult it with someone now?

Reply:
:+1: @gtanzillo who does doc for sources?

Reply:
@agrare @slemrmartin It was decided that documentation for sources would be included in the doc for each application that depends on having a source. I'll work with Ian to make sure this is addressed in the FiFi doc.